lib/views/auth/spree/users/show.html.erb

```diff
@@ -0,0 +1,50 @@
+<div class="account-summary">
+  <div class="account-summary__profile">
+    <h2><%= I18n.t('spree.email') %></h2>
+    <p><%= @user.email %> (<%= link_to I18n.t('spree.edit'), spree.edit_account_path %>)</p>
+  </div>
+
+  <div class="account-summary__orders">
+    <h2><%= I18n.t('spree.my_orders') %></h2>
+    <% if @orders.present? %>
+      <% @orders.each do |order| %>
+        <ul class="account-order">
```

Comment:
Maybe a description list would be more appropriate here.

Link: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

Reply:
👍